<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeLayoutClass</class>
 <widget class="QMainWindow" name="ChangeLayoutClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChangeLayout</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btn_layout_2x2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>2*2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_layout_3x3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>3*3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_layout_1_5">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>1+5</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_w1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string>W1</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_w2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string>W2</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_w3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>160</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string>W3</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_w4">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>160</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string>W4</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_w5">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>160</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string>W5</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_w6">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string>W6</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_w7">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>230</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string>W7</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_w8">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>230</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string>W8</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_w9">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>230</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string>W9</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>649</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ChangeLayout.qrc"/>
 </resources>
 <connections/>
</ui>
